pennylane/templates/embeddings.py

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 r"""
-This module provides quantum circuit architectures that can embed features into a quantum state.
+Embeddings are templates that take features and encode them into a quantum state.
+They can optionally be repeated, and may contain trainable parameters.
```

Comment:
Good to mention that they are often---but not always---found at the start of a circuit?